framework/platform/bpl/uci/cfg/bpl_cfg_uci.cpp

```diff
@@ -265,6 +288,23 @@ int cfg_uci_get_wireless_idx(char *interfaceName, int *rpc_index)
 
             *rpc_index =
                 (*rpc_index >= DUMMY_VAP_OFFSET) ? (*rpc_index - DUMMY_VAP_OFFSET) : *rpc_index;
+
+#elif BEEROCKS_RDKB
+            if (scanf_res < 1 || *rpc_index < 0) {
+                *rpc_index = -1;
+                uci_free_context(ctx);
+                return RETURN_ERR;
+            }
+
+            /* if it is dummy we want the radio index, not the dummy one */
+            if (uci_converter_is_dummy(*rpc_index)) {
+                *rpc_index = dummy_to_radio_index(*rpc_index);
```

Comment:
You could save a line here, but that's just a bonus ;)
```suggestion
                if ((*rpc_index = dummy_to_radio_index(*rpc_index)) == RETURN_ERR) { ... }
```

Reply:
I prefer to not change the function since it's copied from swpal.